spec/models/user_spec.rb

```diff
@@ -87,13 +89,53 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before { events_registration.update_attributes(attended: true) }
```

Comment:
@Ana06 But the preferred Ruby syntax for one-liner blocks is with `{ }` 😕 . And this same syntax is used several times across the file in lines I didn't write.